style.css

```diff
@@ -262,180 +308,152 @@ input[type=submit]:hover{
 	clear: both;
 }
 
-/*--------Home Content Box------------*/
-
-.content_2{
-	box-sizing: border-box;
-	margin: -2vw 5vw 0vw;
-	width: 80%;
-	height: 35vw;
-	/*background-color: #f7f7f7;*/
-	z-index: 2;
-	padding-left: 15px;
-	padding-right: 10px;
-	/*background: rgba(204, 204, 204, 0.7);*/
-	grid-column: 3/5;
-	grid-row: 2;
-}
 /*-------Library Content ----------*/
-.content_padding{
-	padding-top: 0px;	
-}
 
-.content_3{
+.content_5{
```

Comment:
From content_5 and below I have changed the grid columns and rows so they fit in the design better. I have also change the 'px' to 'vw' so they work better with the responsive design.